pkg/bucketeer/model/evaluation_detail.go

```diff
@@ -0,0 +1,60 @@
+package model
+
+type EvaluationDetail[T EvaluationValue] struct {
```

Comment:
For all the public structs, we use the prefix `BKT`.

```suggestion
type BKTEvaluationDetail[T EvaluationValue] struct {
```